website/docs/r/transfer_workflow.html.markdown

```diff
@@ -43,8 +73,8 @@ The following arguments are supported:
 
 #### Copy Step Details
 
-* `destination_file_location` - (Optional) Specifies the location for the file being copied. Use ${Transfer:username} in this field to parametrize the destination prefix by username.
 * `name` - (Optional) The name of the step, used as an identifier.
+* `destination_file_location` - (Optional) Specifies the location for the file being copied. Use ${Transfer:username} in this field to parametrize the destination prefix by username.
```

Comment:
I'd be interested to hear your line of thought for this change. The list of arguments is currently in alphabetical order; is there a reason to consideration moving this line?

Reply:
Initially it felt appropriate as all other steps started with `name`. However, it seems I'd overlooked the alphabetic approach - which I am entirely in favour of.